src/components/TodoList.style.tsx

```diff
@@ -0,0 +1,13 @@
+import { makeStyles } from "@mui/styles";
+
+export const useTodoListStyles = makeStyles((theme) => ({
+  listChecked: {
+    textDecoration: "line-through !important",
```

Comment:
Do we need important here?

Reply:
Because authored CSS is being overridden by the default MUI style.